### PR TITLE
Yin & Yang rating

### PR DESCRIPTION
**Pull request by xithan**
nestle_-_wikirate

### CODE REVIEW

File: db/migrate_cards/20150119162522_add_sourcefile_type.rb [hunk 1]
```diff
@@ -0,0 +1,22 @@
+# -*- encoding : utf-8 -*-
+
+class AddSourcefileType < Card::Migration
+  def up
+    if (source = Card.fetch('Source file'))
+      source.update_attributes! :type_id=>Card::CardtypeID, :codename=>'source_file'
+    else
+      Card.create! :name=>'Source file', :type_id=>Card::CardtypeID, :codename=>'source_file'
+    end
```
Comment (ethn) [inline db/migrate_cards/20150119162522_add_sourcefile_type.rb:9]: wait, so we had already created a "Source file" card for this?

Comment (xithan) [inline db/migrate_cards/20150119162522_add_sourcefile_type.rb:9]: but we added the oxfam data as file and not as source file

File: db/migrate_cards/data/yinyang.json [hunk 2]
```diff
@@ -0,0 +1,168 @@
...
+      {
+        "name": "Topic+Company+*ltype rtype+*structure",
+        "type": "Basic",
+        "content": ""
+      },
+      {
+        "name": "needs polish",
+        "type": "Pointer",
+        "content": "[[yinyang]]\n[[Company+metric sidebar+*type plus right+*structure]]\n[[Company+topic sidebar+*type plus right+*structure]]\n[[metric drag item]]\n[[topic drag item]]\n[[topic details+*right+*structure]]\n[[metric details+*right+*structure]]\n[[style: rating]]\n[[script: rating]]\n[[*upvotes+*empty list]]\n[[*downvotes+*empty list]]\n[[metric value timeline item]]"
+      },
```
Comment (xithan) [inline db/migrate_cards/data/yinyang.json:110]: I collected all new cards that have something to do with the appearance in this pointer

**Comment by xithan** [comment]
tests need the pending wagn/mods PR to pass

**Pull request merged**